runtime/src/main/scala/akka/grpc/internal/PlayRouter.scala

```diff
@@ -65,7 +65,9 @@ import scala.compat.java8.OptionConverters._
    * so therefore not supported.
    */
   final override def withPrefix(prefix: String): Router =
-    if (prefix == this.prefix) this
+    // explicitly stating the full path is fine, we also treat / as identity for convenience so that users do not
+    // _have_ to repeat the service name but can just have a / in the routes file
+    if (prefix == this.prefix || prefix == "/") this
```

Comment:
Discussed with Rich and we're going to make it feel more "play", the `prefix == this.prefix` skipping is something that Rich was not happy with but the `prefix == /` is ok for this logic